ffi/wgpu.h

```diff
@@ -289,11 +289,15 @@ typedef enum {
   WGPUVertexFormat_Int4 = 29,
 } WGPUVertexFormat;
 
+typedef struct WGPUBufferCopyView WGPUBufferCopyView;
+
 /**
  * The internal enum mirrored from `BufferUsage`. The values don't have to match!
  */
 typedef struct WGPUBufferUse WGPUBufferUse;
 
+typedef struct WGPUTextureCopyView WGPUTextureCopyView;
```

Comment:
I don't think these changes are expected – maybe we need to whitelist `WGPUTextureCopyView`, `WGPUBufferCopyView`, and possibly others if they aren't automatically included (`WGPUOrigin3d`)?

Reply:
fixing in https://github.com/gfx-rs/wgpu/pull/669